examples/dropwizard/pom.xml

```diff
@@ -0,0 +1,148 @@
+                                    <jmxFlag>-Djava.awt.headless=true</jmxFlag>
+                                    <jmxFlag>-XX:+UnlockExperimentalVMOptions</jmxFlag>
+                                    <jmxFlag>-XX:+UseCGroupMemoryLimitForHeap</jmxFlag>
+                                    <jmxFlag>-XX:InitialRAMFraction=2</jmxFlag>
+                                    <jmxFlag>-XX:MinRAMFraction=2</jmxFlag>
+                                    <jmxFlag>-XX:MaxRAMFraction=2</jmxFlag>
+                                    <jmxFlag>-XX:+UseG1GC</jmxFlag>
+                                    <jmxFlag>-XX:MaxGCPauseMillis=100</jmxFlag>
+                                    <jmxFlag>-XX:+UseStringDeduplication</jmxFlag>
+                                </jvmFlags>
+                            </container>
+                        </configuration>
+                        <executions>
+                            <execution>
+                                <id>dockerBuild</id>
+                                <goals>
+                                    <goal>dockerBuild</goal>
+                                </goals>
+                                <phase>package</phase>
+                            </execution>
+                        </executions>
+                    </plugin>
+                </plugins>
+            </build>
+        </profile>
+    </profiles>
+
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>io.dropwizard</groupId>
+                <artifactId>dropwizard-bom</artifactId>
+                <version>${dropwizard.version}</version>
+                <type>pom</type>
+                <scope>import</scope>
+            </dependency>
+        </dependencies>
+    </dependencyManagement>
+
+    <dependencies>
+        <dependency>
+            <groupId>io.dropwizard</groupId>
+            <artifactId>dropwizard-core</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>de.thomaskrille</groupId>
+            <artifactId>dropwizard-template-config</artifactId>
+            <version>${dropwizard-template-config.version}</version>
+        </dependency>
+    </dependencies>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>exec-maven-plugin</artifactId>
+                <version>1.6.0</version>
+                <executions>
+                    <execution>
+                        <id>java</id>
+                        <goals>
+                            <goal>java</goal>
+                        </goals>
+                    </execution>
+                </executions>
+                <configuration>
+                    <executable>java</executable>
+                    <mainClass>${mainClass}</mainClass>
+                    <arguments>
+                        <argument>server</argument>
+                        <argument>${dropwizard.server.config}</argument>
```

Comment:
Since the `jib` profile is activated by default, this is overridden to `/app/config/dropwizard.yml`, which makes `./mvnw exec:java` fail.

Reply:
You could put this file in `src/main/resources` and then reference it in the Jib `<args>` as `/app/resources/dropwizard.yml`.

Reply:
That makes more sense, actually, because it's resources _for the application_, not for Jib.